examples/pytorch/text-generation/run_generation.py

```diff
@@ -248,6 +381,18 @@ def main():
     else:
         input_ids = encoded_prompt
 
+    if args.jit:
+        jit_input_texts = ["jit"]
+        jit_inputs = prepare_jit_inputs(jit_input_texts, model, tokenizer)
+        torch._C._jit_set_texpr_fuser_enabled(False)
+        model.config.return_dict = False
+        traced_model = torch.jit.trace(model, jit_inputs, strict=False)
+        traced_model = torch.jit.freeze(traced_model.eval())
+        traced_model(*jit_inputs)
+        traced_model(*jit_inputs)
```

Comment:
> Thanks for this contribution! 🔥
> 
> Have you run the script with and without the `--jit` flag to confirm it runs as expected?

Yes, the example runs as usual without --jit.



> Why is this line run twice? I'm not super familiar with `torch.jit` so apologies if I'm missing something.

I have tested it on A100 and found that the first two forwards are very slow, mainly because the first forward contains the operation of inserting profiling nodes and the second forward contains the operation of fusion. Refer to [jit](https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp#L655).